[line:39]2020-06-26 15:52:36,320-INFO-logger.py-case-日志信息:测试开始
[line:40]2020-06-26 15:52:36,320-ERROR-logger.py-case-日志信息:测试报错
[line:42]2020-06-26 15:52:36,320-INFO-logger.py-case-日志信息:测试结束
[line:24]2020-06-26 16:44:31,381-INFO-test_login.py-test_login-日志信息:准备测试前置
[line:29]2020-06-26 16:44:31,384-INFO-test_login.py-test_login-日志信息:开始测试：正常登陆
[line:39]2020-06-26 16:44:32,158-INFO-test_login.py-test_login-日志信息:结束测试：正常登陆
[line:29]2020-06-26 16:44:32,159-INFO-test_login.py-test_login-日志信息:开始测试：密码错误
[line:39]2020-06-26 16:44:32,831-INFO-test_login.py-test_login-日志信息:结束测试：密码错误
[line:29]2020-06-26 16:44:32,832-INFO-test_login.py-test_login-日志信息:开始测试：手机号错误
[line:39]2020-06-26 16:44:32,931-INFO-test_login.py-test_login-日志信息:结束测试：手机号错误
[line:43]2020-06-26 16:44:32,932-INFO-test_login.py-test_login-日志信息:测试后置处理
[line:24]2020-06-26 16:54:16,026-INFO-test_login.py-API_8.testcases.test_login-日志信息:准备测试前置
[line:29]2020-06-26 16:54:16,026-INFO-test_login.py-API_8.testcases.test_login-日志信息:开始测试：正常登陆
[line:39]2020-06-26 16:54:16,768-INFO-test_login.py-API_8.testcases.test_login-日志信息:结束测试：正常登陆
[line:29]2020-06-26 16:54:16,769-INFO-test_login.py-API_8.testcases.test_login-日志信息:开始测试：密码错误
[line:39]2020-06-26 16:54:17,432-INFO-test_login.py-API_8.testcases.test_login-日志信息:结束测试：密码错误
[line:29]2020-06-26 16:54:17,432-INFO-test_login.py-API_8.testcases.test_login-日志信息:开始测试：手机号错误
[line:39]2020-06-26 16:54:17,532-INFO-test_login.py-API_8.testcases.test_login-日志信息:结束测试：手机号错误
[line:43]2020-06-26 16:54:17,533-INFO-test_login.py-API_8.testcases.test_login-日志信息:测试后置处理
[line:24]2020-06-26 17:18:19,045-INFO-test_login.py-test_login-日志信息:准备测试前置
[line:29]2020-06-26 17:18:19,046-INFO-test_login.py-test_login-日志信息:开始测试：正常登陆
[line:39]2020-06-26 17:18:19,816-INFO-test_login.py-test_login-日志信息:结束测试：正常登陆
[line:29]2020-06-26 17:18:19,816-INFO-test_login.py-test_login-日志信息:开始测试：密码错误
[line:39]2020-06-26 17:18:20,505-INFO-test_login.py-test_login-日志信息:结束测试：密码错误
[line:29]2020-06-26 17:18:20,505-INFO-test_login.py-test_login-日志信息:开始测试：手机号错误
[line:39]2020-06-26 17:18:20,628-INFO-test_login.py-test_login-日志信息:结束测试：手机号错误
[line:43]2020-06-26 17:18:20,629-INFO-test_login.py-test_login-日志信息:测试后置处理
[line:24]2020-06-26 17:49:45,488-INFO-test_login.py-test_login-日志信息:准备测试前置
[line:29]2020-06-26 17:49:45,489-INFO-test_login.py-test_login-日志信息:开始测试：正常登陆
[line:39]2020-06-26 17:49:46,249-INFO-test_login.py-test_login-日志信息:结束测试：正常登陆
[line:29]2020-06-26 17:49:46,249-INFO-test_login.py-test_login-日志信息:开始测试：密码错误
[line:39]2020-06-26 17:49:46,924-INFO-test_login.py-test_login-日志信息:结束测试：密码错误
[line:29]2020-06-26 17:49:46,924-INFO-test_login.py-test_login-日志信息:开始测试：手机号错误
[line:39]2020-06-26 17:49:47,024-INFO-test_login.py-test_login-日志信息:结束测试：手机号错误
[line:43]2020-06-26 17:49:47,025-INFO-test_login.py-test_login-日志信息:测试后置处理
